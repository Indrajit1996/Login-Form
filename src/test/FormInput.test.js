import React from 'react';
import { create } from "react-test-renderer";
import FormInput from '../presenters/FormInput';

describe("Form Input component", () => {
    test("Matches the snapshot", () => {
      const formInput = create(<FormInput />);
      expect(formInput.toJSON()).toMatchSnapshot();
    });
});