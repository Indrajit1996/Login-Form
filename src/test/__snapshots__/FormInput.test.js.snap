// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component Matches the snapshot 1`] = `
Array [
  <label />,
  <input
    className=""
    style={null}
    type="text"
  />,
]
`;

exports[`Form Input component Matches the snapshot 1`] = `
Array [
  <label />,
  <input
    className=""
    style={null}
    type="text"
  />,
]
`;
